src/EditModel/EditDataEntryForm/processFormData.js

```diff
@@ -0,0 +1,102 @@
+import log from 'loglevel';
+
+// TODO: write tests for processFormData
```

Comment:
`processFormData` would benefit from some tests but I wasn't sure what kind of input/output would be useful to test